<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumberKeyBoardClass</class>
 <widget class="QWidget" name="NumberKeyBoardClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NumberKeyBoard</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(183, 230, 255);</string>
  </property>
  <widget class="QPushButton" name="pbNumber8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>84</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/image/button_key_num.png);
</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbNumber9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>84</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/image/button_key_num.png);
</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDot">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>84</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/image/button_key_num.png);
</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDisplayNumber">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbNumber6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>84</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/image/button_key_num.png);
</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbNumber4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>84</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/image/button_key_num.png);</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbNumber7">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>84</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/image/button_key_num.png);
</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbNumber5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>84</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/image/button_key_num.png);
</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbNumber0">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>84</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/image/button_key_num.png);
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbNumber3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>84</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/image/button_key_num.png);</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbNumber1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>84</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/image/button_key_num.png);
</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbNumber2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>84</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/image/button_key_num.png);</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbBackspace">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>86</width>
     <height>77</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/image/button_key_backenter.png);
</string>
   </property>
   <property name="text">
    <string>退格</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbEnter">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>86</width>
     <height>77</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/image/button_key_backenter.png);
</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbMinus">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>84</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/image/button_key_num.png);
</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
